src/main/scala/org/ergoplatform/appkit/ergotool/AddressCmd.scala

```diff
@@ -34,18 +35,19 @@ case class AddressCmd
 
 /** Descriptor and parser of the `address` command. */
 object AddressCmd extends CmdDescriptor(
-  name = "address", cmdParamSyntax = "testnet|mainnet <mnemonic>",
+  name = "address", cmdParamSyntax = "testnet|mainnet",
   description = "return address for a given mnemonic and password pair") {
 
   override val parameters: Seq[CmdParameter] = Array(
     CmdParameter("network", NetworkPType,
       "[[NetworkType]] of the target network for which the address should be generated"),
-    CmdParameter("mnemonic", SecretStringPType,
+    CmdParameter("mnemonic", "Mnemonic", SecretStringPType,
       """secret phrase which is used to generate (private, public) key pair, of which
-        |public key is used to generate the [[Address]]""".stripMargin),
+        |public key is used to generate the [[Address]]""".stripMargin, None, 
+        Some(PasswordInput), None),
```

Comment:
@greenhat PasswordInput will be invisible to the user and hard to input correctly. I suggest to use DefaultParameterInput

Reply:
Thanks! Fixed.